pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/patch/Patch.java

```diff
@@ -142,12 +142,13 @@ public static boolean patch(@NonNull String id, @NonNull String user, @NonNull N
                 if (result == 1) {
                     logger.info("Updated " + dbElement.getTableName() + " record w/ id " + id);
                 } else {
-                    throw new RuntimeException(
-                            "Error updating " + dbElement.getTableName() + " record of w/ id " + id + " it was updating " + result
-                                    + " rows");
+                    throw new RuntimeException("Error updating " + dbElement.getTableName() + " with "
```

Comment:
Sorry, I missed this the first time around...
We want to replace `RuntimeException` with something more meaningful that indicates how to generate the response. We introduced DSMBadRequestException (400) and DsmInternalError (500) for that purpose. In this case DsmInternalError makes sense.
Also, `String.format` would make this code more readable.